libfwupd/fwupd-security-attr.h

```diff
@@ -109,6 +110,7 @@ typedef enum {
 	FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND,	  /* Since: 1.5.0 */
 	FWUPD_SECURITY_ATTR_RESULT_SUPPORTED,	  /* Since: 1.5.0 */
 	FWUPD_SECURITY_ATTR_RESULT_NOT_SUPPORTED, /* Since: 1.5.0 */
+	FWUPD_SECURITY_ATTR_RESULT_MISSING_DATA,  /* Since: 1.8.0 */
```

Comment:
Maybe this is a *flag* more than a result; it's not saying "this is the answer" it's saying "not enough data to get an answer"

Reply:
updated